Habitica/src/main/java/com/habitrpg/android/habitica/utils/ContentDeserializer.kt

```diff
@@ -101,6 +94,20 @@ class ContentDeserializer : JsonDeserializer<ContentResult> {
                 result.spells.add(skill)
             }
         }
+
+        if (obj.has("special")) {
+            for ((classname, value) in obj.getAsJsonObject("special").entrySet()) {
+                val skillItem = value.asJsonObject

```

Comment:
Is there a reason against using `context.deserialize(obj.get("special"), object : TypeToken<RealmList<SpecialItem>>() {}.type)` like we do for other entries?